### PR TITLE
feat: drawRectangle

### PR DESCRIPTION
**Pull request by wadjih-bencheikh18**

### CODE REVIEW

**Comment by wadjih-bencheikh18** [comment]
does using `paintline`  make the function slower?

**Comment by targos** [comment]
Currently yes, but it will be better when https://github.com/image-js/image-js-typescript/issues/89 is fixed.
In the future, I think we will need to implement more low-level and performant drawing functions that can be then used by drawLine, drawRectangle, etc.

**Comment by wadjih-bencheikh18** [comment]
>In the future, I think we will need to implement more low-level and performant drawing functions that can be then used by drawLine, drawRectangle, etc.

Could you please assign to me this task

**Comment by targos** [comment]
There are still some "paint" to change. See https://github.com/image-js/image-js-typescript/pull/84/files

**Comment by wadjih-bencheikh18** [comment]
> There are still some "paint" to change. See https://github.com/image-js/image-js-typescript/pull/84/files

fixed, i saw some paint in draw line but it's already fixed in https://github.com/image-js/image-js-typescript/pull/83

File: src/draw/__tests__/drawRectangle.test.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    const out = new IJS(2, 3);
+    const image = testUtils.createRgbImage([
+      [100, 150, 200, 100, 150, 0],
+      [100, 200, 5, 3, 200, 0],
+      [150, 200, 255, 6, 150, 0],
+    ]);
+    const start = { row: 1, column: 0 };
+    const width = 3;
+    const height = 3;
+    const expected = image.drawRectangle(start, width, height, {
```
Comment (lpatiny) [inline src/draw/__tests__/drawRectangle.test.ts:57]: Please use only one object parameter that contains the following properties:
- row: default 0
- column: default 0
- width: default image.width
- height: default image.height
- color: default [0,0,0] (depends of the kind of image as you did)
- fill: default empty

**Review by lpatiny — CHANGES_REQUESTED** [review]

File: src/draw/__tests__/drawRectangle.test.ts (hunk 1 above)
Comment (wadjih-bencheikh18) [reply src/draw/__tests__/drawRectangle.test.ts:57]: If default color is black how could we create rectangle with fill and no color 
(i suggest adding color "none")

Comment (targos) [reply src/draw/__tests__/drawRectangle.test.ts]: `color: 'none'` SGTM

Comment (wadjih-bencheikh18) [reply src/draw/__tests__/drawRectangle.test.ts]: done

**Pull request merged**